package.json

```diff
@@ -37,6 +36,9 @@
     "ts-node": "^10.9.2",
     "typescript": "^5.5.2"
   },
+  "peerDependencies": {
+    "@balena/sbvr-types": "^8.0.0, ^9.0.0"
```

Comment:
Can you add 7.x to this as well please since it'll work fine and is still what is actually used by pinejs currently

Reply:
Although it won't be 7.0.0, I'm not sure what the minimum version was to expose the bits we need but it should match that

Reply:
@Page- As of writing this balena-api is using pinejs v17.2.1, which has sbvr-types at v7.1.3. For now I've set the minimum required v7 here to `^7.1.0` as that's the last minor of v7 and exports the typescript read/write types for each sbvr type, which I assume is what you mean when you say "expose the bits we need".